cli/app.go

```diff
@@ -2935,6 +2941,16 @@ This won't work unless you have an existing installation of our GitHub app on yo
 	# Restart a module running on your local viam server, by name, without building or reconfiguring.
 	viam module reload --restart-only --id viam:python-example-module
 
+	# Use cloudbuild to build the tar.gz that will be copied to the part for hot reloading.
+	viam module reload --cloud-build
+
+	# Run viam module reload on a mac and use the downloaded viam.json file instead of --part-id
+	viam module reload --cloud-config ~/Downloads/viam-mac-main.json
+
+	# Specify which component/service to add to the config along with the module (the API is
+	# automatically looked up from meta.json)
+	viam module reload --model-name acme:module-name:mybase
```

Comment:
love this 'add a component with defaults' workflow as a way to shorten the # of steps for a new user to get started

Reply:
model-name isn't actually being used at all right now! 😅 thanks for flagging. I will either implement or remove and file a follow-up ticket.